hail_scripts/reference_data/constants.py

```diff
@@ -0,0 +1 @@
+GCS_PREFIX = "gs://seqr-reference-data/GRCh{genome_version}/v03/{environment}"
```

Comment:
With how this is written now, we'll get both a `dev` and a `prod` prefix here.  I think `dev` is nice but `prod` might be unnecessary.  The implementation will be slightly trickier though if we want to have `prod` mean `''` though.  Anyone else have thoughts?

Reply:
I don't want us to pay to maintain dev versions of files, and any downstream dev versions of the pipeline should for the most part be using the main production files. If we want to support writing test versions of these files I would say we should be writing them to `gs://seqr-scratch-temp`

Reply:
👍 I can just make the paths the same for `dev`/`prod` and have `dev` -> `gs://seqr-scratch-temp` and `prod` -> `gs://seqr-reference-data`.

fwiw you can setup a retention policy on path prefixes: https://cloud.google.com/blog/products/storage-data-transfer/understanding-cloud-storages-new-prefix-and-suffix-lifecycle-rules but that can get kinda messy.